umbraco/Umbraco-CMS
# v9: 11419 Remove Azure gallery from build.ps1

Fixes: https://github.com/umbraco/Umbraco-CMS/issues/11419

# Notes
- Removed Azure gallery from build.ps1 script
- Changed order of itemgroups in csproj template
this order is switched, because if you installed a package for example `Umbraco.TheStarterKit` it would end up in the itemgroup with `Condition="'$(OS)' == 'Windows_NT'"` this would then make it so your packages would only show in a windows install, which is not ideal

# How to test
- Run build.ps1 - should have no errors
- Navigate to build.out and install the Umbraco.Templates with `dotnet new -i ./Umbraco.Templates.9.1.0.nupkg`
- Install a new umbraco project with `dotnet new umbraco -n MyTestProj`
- Install the starter kit in your new project with `dotnet add package Umbraco.TheStarterKit`
- Open the csproj and ensure the `<PackageReference Include="Umbraco.TheStarterKit" Version="9.0.0" />` ends up in the right itemgroup
